pkg/component/grpc_test.go

```diff
@@ -71,6 +71,10 @@ func (as *asImplementation) DownlinkQueueList(context.Context, *ttnpb.EndDeviceI
 	return nil, errors.New("not implemented")
 }
 
+func (as *asImplementation) GetMQTTConnectionInfo(context.Context, *ttnpb.ApplicationIdentifiers) (*ttnpb.MQTTConnectionInfo, error) {
```

Comment:
We should probably start using the `ttnpb.UnimplementedXXXServer`s instead of writing all this ourselves (here, `asImplementation` could embed a `ttnpb.UnimplementedAppAsServer`). Not really necessary to do it in this PR, but good to change it at some point.